llvm/utils/TableGen/Common/CodeGenDAGPatterns.cpp

```diff
@@ -3392,8 +3392,10 @@ static bool HandleUse(TreePattern &I, TreePatternNodePtr Pat,
   Record *Rec;
   if (Pat->isLeaf()) {
     DefInit *DI = dyn_cast<DefInit>(Pat->getLeafValue());
-    if (!DI)
+    if (!DI) {
       I.error("Input $" + Pat->getName() + " must be an identifier!");
```

Comment:
Mind adding a test that covers this error message? ubsan'd builds of llvm would have caught this if we had one.

Reply:
Unfortunately I don't have input or compiled target to reproduce error - the bug was found by static analysis